aten/src/ATen/native/DispatchStub.cpp

```diff
@@ -114,18 +115,27 @@ void* DispatchStubImpl::choose_cpu_impl(
 ) {
   auto capability = static_cast<int>(get_cpu_capability());
   (void)capability;
+#ifdef HAVE_AVX512_CPU_DEFINITION
+  if (capability >= static_cast<int>(CPUCapability::AVX512)) {
+    // nansum & sum currently don't have AVX512 kernels, as even with AVX2,
+    // they have poor accuracy with Float16 (GH issues 59415 & 59489), and
+    // even poorer with AVX512. Until their accuracy is improved, they would
+    // be dispatched to AVX2 kernels. Quantization kernels have also been
+    // disabled on Windows for AVX512 because some of their tests are flaky.
+    if C10_UNLIKELY(!AVX512) {
+      TORCH_INTERNAL_ASSERT(AVX2, "DispatchStub: missing AVX2 kernel");
```

Comment:
```suggestion
      TORCH_INTERNAL_ASSERT(AVX2, "DispatchStub: missing AVX512 kernel");
```

Reply:
This one's intended. Some kernels, such as the Quantization kernels on Windows aren't being dispatched with AVX512 at this point. In that case, their AVX2 kernels are being dispatched.